server/lib/queries.js

```diff
@@ -34,7 +34,7 @@ const generateFXConversionSQL = aggregate => {
   return sql;
 };
 
-const getPublicHostsByTotalCollectives = args => {
+const getPublicHostsByTotalCollectives = async args => {
```

Comment:
I changed this query because it was not mapping results to models, thus the `collective.canApply` call was failing in graphql enpoint. It will alto fix the `total` number returned, which was invalid.